data.sql

```diff
@@ -162,4 +162,8 @@ FULL JOIN specializations ON specializations.vets_id = vets.id
 WHERE vets.name = 'Maisy Smith' 
 GROUP BY S 
 ORDER BY CNT 
-DESC LIMIT 1;
\ No newline at end of file
+DESC LIMIT 1;
+
+--Performance Improvement--
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/215528319-458fe667-2826-4d3e-a05b-ea630a054b60.png)
- [ ] Kindly add some Future Features of your application to make your readme file descriptive and to create excitement for the viewers of your repository :+1: